data/korath weapons.txt

```diff
@@ -185,9 +185,9 @@ effect "repeater impact"
 	sprite "effect/repeater impact"
 		"no repeat"
 		"frame rate" 12
+	"random velocity" .6
```

Comment:
I'd order it random angle -> random spin -> random velocity for effects instead of moving random velocity to where it would typically be for weapons, partially because we also have the `"velocity scale"` attribute at the bottom. 

The rest of these changes seem fine to me; doesn't hurt anything, and it's better to keep attributes consistently ordered (although I wouldn't go crazy over white space).